test/e2e/core/selectAll.spec.js

```diff
@@ -22,8 +22,8 @@ describe('Core.selectAll', () => {
 
     selectCells([[1, 1, 2, 2], [2, 2, 4, 4]]);
 
-    hot.view.wt.wtTable.holder.scrollTop = 150;
-    hot.view.wt.wtTable.holder.scrollLeft = 150;
+    hot.view.wt.wtTable.holder.scrollTop = 155;
```

Comment:
This change is motivated by the different scrollbar width between windows and macOS, isn't it?. I wonder if we can calculate this value based on available space, rows heights etc. as in test specs for selection and table.

Reply:
I have pushed a soution with even cleaner solution in the commit a5850d17ab1120d4614c2636ab82ac759063191e.